test/blocks/caas/utils.test.js

```diff
@@ -164,7 +164,7 @@ describe('getConfig', () => {
           cardTitleAccessibilityLevel: 6,
           prettyDateIntervalFormat: '{ddd}, {LLL} {dd} | {timeRange} {timeZone}',
           totalResultsText: '{total} Results',
-          title: 'My Awesome Title',
+          title: '',
```

Comment:
It looks like this only tests one of your options for the title (AKA "leave empty"). Do you want to add tests for the other two?

Reply:
@meganthecoder thanks for checking. :)
Other collection title options are tested here:
https://github.com/adobecom/milo/blob/e4455917032bfa9a2e89787ae23de8d885d6e2ae/test/blocks/caas/utils.test.js#L73-L96